src/app/routes/connector-ui/dashboard-page/dashboard-page/dashboard-page-data.service.ts

```diff
@@ -151,9 +153,15 @@ export class DashboardPageDataService {
     );
 
     const colorsByState = new Map<string, string>();
-    colorsByState.set('IN_PROGRESS', '#7eb0d5');
-    colorsByState.set('ERROR', '#fd7f6f');
-    colorsByState.set('COMPLETED', '#b2e061');
+    if (this.activeFeatureSet.hasMdsFields()) {
+      colorsByState.set('IN_PROGRESS', '#FFFF00');
+      colorsByState.set('ERROR', '#FFA500');
+      colorsByState.set('COMPLETED', '#9ACD32');
+    } else {
+      colorsByState.set('IN_PROGRESS', '#7eb0d5');
+      colorsByState.set('ERROR', '#fd7f6f');
+      colorsByState.set('COMPLETED', '#b2e061');
+    }
```

Comment:
We should have three additional config properties "chartColorNeutral, chartColorSuccess, chartColorFailure" in the AppConfig and EdcUiThemeConfig, then specify values in `edc-ui-theme-data.ts`.